src/app/portal/models/annotations/annotations.type.ts

```diff
@@ -7,5 +7,18 @@ export type Annotations = {
   hearingDate: DateTime;
   annotationTs: DateTime;
   annotationText: string;
-  annotationDocuments: AnnotationDocument[];
-};
+} & AnnotationDocument;
+
+// Example object
+// {
+//     "annotationId": 1,
+//     "hearingId": 2,
+//     "hearingDate": "2023-12-14T00:00:00.000Z",
+//     "annotationTs": "2023-12-15T12:00:00.000Z",
+//     "annotationText": "A summary notes of this annotation...",
+//     "annotationDocumentId": 1,
+//     "fileName": "Annotation.doc",
```

Comment:
Is Mr McUserFace object supposed to be checked in 😂

Reply:
the object's been removed now